migration/migration.go

```diff
@@ -251,6 +251,9 @@ func BootstrapWorkItemLinking(ctx context.Context, linkCatRepo *link.GormWorkIte
 	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkPlannerItemRelated, "One planner item or a subtype of it relates to another one.", link.TopologyNetwork, "relates to", "relates to", workitem.SystemPlannerItem, workitem.SystemPlannerItem, link.SystemWorkItemLinkCategorySystem); err != nil {
 		return err
 	}
+	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkEpicRelatedUserStory, "Epic Includes User story and vice versa", link.TopologyNetwork, "includes", "included in", workitem.SystemEpic, workitem.SystemUserStory, link.SystemWorkItemLinkCategorySystem); err != nil {
```

Comment:
The sentence is not following proper capitalization rules. Ref. http://www.grammarbook.com/punctuation/capital.asp